mkdocs.yml

```diff
@@ -1,21 +1,10 @@
 ---
-site_name: 'Rocky Linux Documentation'
+site_name: 'Documentation'
 site_url: 'https://docs.rockylinux.org'
 docs_dir: 'docs/docs'
```

Comment:
If there is no technical reason why we have docs/docs, would like to reduce this to :

docs_dir: 'docs'

Reply:
This isn't something user-facing -- it sets the folder for the documentation root. So there _is_ still a technical reason for that one

Reply:
@hbjydev   -  Yes, you are correct, it isn't user facing.  
We have successfully implemented the current setup without docs/docs .  A  single docs is simpler and cleaner.   What is the technical reason why the github docs. repo can't be cloned into/under  a single docs ?

Reply:
@wsoyinka on the point of doc/docs I would agree with @hbjydev, it has no bearing impact to documentation authors/editors (they don't even see it) so no issue to them.

Part of this is down to the fact that there are two repos - let's call them outer and inner. The outer is where the server part `mkdocs.yml` lives in and the inner is the documentation site. Now the documentation site, if one looks at the documentation repo, has its root with various stuff one of which is the folder for documentation source MD files.

The outer has to have the inner as a containing folder, it was called docs. So, from `mkdocs.yml` to get to doc source files it has to navigate to the inner root (called docs) and then to the docs folder in it.

So basically, again the authors never see all this so it's not really a concern. I think it's fine to leave it as it is.